ibis/backends/base/sql/alchemy/client.py

```diff
@@ -329,3 +329,120 @@ def _sqla_table_to_expr(self, table):
     def version(self):
         vstring = '.'.join(map(str, self.con.dialect.server_version_info))
         return parse_version(vstring)
+
+    def insert(
+        self,
+        to_table_name: str,
+        database: Optional[str] = None,
+        data_obj=None,
+        from_table_name: Optional[str] = None,
+        if_exists: Optional[str] = 'append',
+    ) -> None:
+        """
+        Insert the given data to a table in backend.
+
+        Parameters
+        ----------
+        to_table_name : string
+            name of the table to which data needs to be inserted
+        database : string, optional
+            name of the attached database that the table is located in.
+        data_obj : pd.Dataframe
+            data_obj is the dataframe containing data which needs to be
+            inserted to to_table_name
+        from_table_name : string, optional
+            name of the table from which data needs to be inserted
+        if_exists : {'append', 'fail', 'replace'}
+            With append, the rows will be added to the table. With fail,
+            an exception will be raised if the table exists. With replace,
+            if the table exists, it'll be dropped, and recreated before
+            inserting the rows.
+
+        Raises
+        -------
+        ValueError
+            You must pass either data_obj (pandas Dataframe) or
+            from_table_name (table name to insert data from)
+
+            Sorry, can't insert from both the data_obj (dataframe)
+            and the from_table_name (table). Please use only one
+            parameter.
+
+        RuntimeError
+            The table already exists
+
+        TypeError
+            No operation is being performed. Either the data_obj
+            parameter is not a pandas dataframe or the
+            from_table_name parameter is not of string datatype.
+
+        """
+
+        if data_obj is None and from_table_name is None:
+            raise ValueError(
+                'You must pass either data_obj (pandas Dataframe)'
+                ' or from_table_name (table name to insert data from)'
+            )
+
+        if isinstance(data_obj, pd.DataFrame) and isinstance(
+            from_table_name, str
+        ):
+            raise ValueError(
+                "Sorry, can't insert from both the data_obj (dataframe)"
```

Comment:
use a more declarative language. e.g.

`Cannot insert from both....` 

also in-line the paramters via f-strings from what was passed (if you its a table name for example)

Reply:
Done!